src/Vegetation/component_models.jl

```diff
@@ -97,16 +97,31 @@ function initialize_auxiliary(
 end
 
 """
-     ClimaLSM.make_rhs(component::AbstractCanopyComponent, canopy)
+     ClimaLSM.make_compute_exp_tendency(component::AbstractCanopyComponent, canopy)
 
-Creates the rhs!(dY,Y,p,t) function for the canopy `component`.
+Creates the compute_exp_tendency!(dY,Y,p,t) function for the canopy `component`.
 
 Since component models are not standalone models, other information
-may be needed and passed in (via the `canopy` model itself). 
-The right hand side for the entire canopy model can make use of 
+may be needed and passed in (via the `canopy` model itself).
+The right hand side for the entire canopy model can make use of
 these functions for the individual components.
 """
-function ClimaLSM.make_rhs(::AbstractCanopyComponent, canopy)
-    function rhs!(dY, Y, p, t) end
-    return rhs!
+function ClimaLSM.make_compute_exp_tendency(::AbstractCanopyComponent, canopy)
+    function compute_exp_tendency!(dY, Y, p, t) end
+    return compute_exp_tendency!
+end
+
+"""
+     ClimaLSM.make_compute_imp_tendency(component::AbstractCanopyComponent, canopy)
+
+Creates the compute_imp_tendency!(dY,Y,p,t) function for the canopy `component`.
+
+Since component models are not standalone models, other information
+may be needed and passed in (via the `canopy` model itself).
+The right hand side for the entire canopy model can make use of
+these functions for the individual components.
+"""
+function ClimaLSM.make_compute_imp_tendency(::AbstractCanopyComponent, canopy)
```

Comment:
we may be able to avoid adding this. If we call make_imp_tendency(c::Canopy), and there is no method defined for it (which currently there is not, because it is all explicit), we wont ever call make_imp_tendency for the components.

Reply:
Good point! I'll remove it.